android/build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
     ext {
         buildToolsVersion = "29.0.5"
         googlePlayServicesVersion = "12.0.1"
-        minSdkVersion = 16
+        minSdkVersion = 21
         compileSdkVersion = 29
         targetSdkVersion = 29
```

Comment:
API level requirement `Android 11 (API level 30)`. November 1, 2021: Required for app updates
https://support.google.com/googleplay/android-developer/answer/9859152?visit_id=637785872990684562-2732748939&rd=1#targetsdk&zippy=%2Ctarget-api-level-requirements-for-play-console

Reply:
Bumped!